README.md

```diff
@@ -307,6 +307,57 @@ $ sudo systemctl start station
 $ sudo systemctl status station
 ```
 
+## API
+
+### `Core({ cacheRoot?: String, stateRoot?: String })`
+
+Returns `Core`.
+
+### `Core#logs.get(module?: String)`
+
+Returns `Promise<String>`.
+
+### `Core#logs.follow(module?: String)`
+
+Returns `AsyncGenerator<String>`.
+
+### `Core#activity.get()`
+
+Returns `Promise<[ActivityEvent]>`.
```

Comment:
`[ActivityEvent]` means an array containing exactly on `ActivityEvent` item.

You should use `ActivityEvent[]` instead. (Alternatively, `Array<ActivityEvent>` should work too, but it's too verbose to my taste.)